src/test/java/elizCurtnazar/tests/DemoQaManager.java

```diff
@@ -0,0 +1,167 @@
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DemoQaManager {
+    private String url;
+    private static ChromeDriver driver = null;
+    String fullName = "Curtnazar Eliz";
+    String email = "eliz_rts_91@hotmail.com";
+    String firstName = "Eliz";
+    String lastName = "Curtnazar";
+    //int age = 31;
+    String age = "31";
+    String salary = "1800";
+    String department = "Marketing";
+
+
+    DemoQaManager(String url, ChromeDriver driver) {
+        this.url = url;
+        this.driver = driver;
+        driver.get(url);
+    }
+
+    public static int selectTabElement() {
+        List<WebElement> tabElement = driver.findElements(By.xpath("/html/body/div[2]/div/div/div[2]/div[1]/div/div/div[1]/div/ul/li[1]/span"));
+        List<String> nameTab = new ArrayList<>();
+        for (WebElement elem : tabElement) {
+            nameTab.add(elem.getText());
+        }
+        System.out.println(nameTab);
+        WebElement textBox = tabElement.get(0);
+        textBox.click();
+        return tabElement.size();
+
+    }
+
+    public void selectTextBox() {
+
+        WebElement textBox = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[1]/div/div/div[1]/div/ul/li[1]/span"));
+        textBox.click();
+        WebElement fullNameTextBox = driver.findElement(By.id("userName"));
+        fullNameTextBox.click();
+        fullNameTextBox.clear();
+        fullNameTextBox.sendKeys(fullName);
+        WebElement emailTextBox = driver.findElement(By.id("userEmail"));
+        emailTextBox.click();
+        emailTextBox.clear();
+        emailTextBox.sendKeys(email);
+        WebElement submitButton = driver.findElement(By.id("submit"));
+        submitButton.sendKeys(Keys.ENTER);
+
+    }
+
+    public void selectCheckBox() {
+        WebElement checkBox = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[1]/div/div/div[1]/div/ul/li[2]/span"));
+        checkBox.click();
+        WebElement clickBoxHome = driver.findElement(By.cssSelector("span.rct-title"));
+        clickBoxHome.click();
+        WebElement textAfisat = driver.findElement(By.id("result"));
+        textAfisat.getText();
+        System.out.println(textAfisat.getText());
+
+    }
+
+    public void selectRadioButton() {
+        WebElement radioButton = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[1]/div/div/div[1]/div/ul/li[3]"));
+        radioButton.click();
+        WebElement question = driver.findElement(By.cssSelector(".mb-3"));
+        question.getText();
+        System.out.println(question.getText());
+        WebElement yesAnswer = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[2]/div[2]/div[2]/label"));
+        yesAnswer.click();
+        WebElement answer = driver.findElement(By.cssSelector(".mt-3"));
+        answer.getText();
+        System.out.println(answer.getText());
+    }
+
+
+    public void WebTablesLink(String link) {
```

Comment:
Please use camelCase when naming methods as per Java conventions. Check it for every method that you wrote. Thanks!